docs/API.md

```diff
@@ -824,6 +834,115 @@ __Example__
 
   ```
 
+  <a name="setBucketNotification"></a>
+### setBucketNotification(String bucketName, NotificationConfiguration notificationConfiguration)
+`public void setBucketNotification(String bucketName, NotificationConfiguration notificationConfiguration)`
+
+Set bucket notification configuration.
+
+[View Javadoc](http://minio.github.io/minio-java/io/minio/MinioClient.html#setBucketNotification-java.lang.String-io.minio.messages.NotificationConfiguration-)
+
+__Parameters__
+
+|Param   | Type   | Description  |
+|:--- |:--- |:--- |
+| ``bucketName``  | _String_  | Name of the bucket.  |
+| ``notificationConfiguration`` | _NotificationConfiguration_ | Notification configuration to be set. |
+
+| Return Type	  | Exceptions	  |
+|:--- |:--- |
+|  None  | Listed Exceptions: |
+|        | ``InvalidBucketNameException`` : upon invalid bucket name. |
+|        | ``InvalidObjectPrefixException`` : upon invalid object prefix. |
+|        | ``NoSuchAlgorithmException`` : upon requested algorithm was not found during signature calculation.  |
+|        | ``InsufficientDataException`` : Thrown to indicate that reading given InputStream gets EOFException before reading given length. |
```

Comment:
The same as the previous comment.
```
This is documentation, it needs to be meaningful text imho.
So, what is `given InputStream` from the perspective of the end-user? 
Why do we say `given`? This is not an input parameter end-user provides.
We should not reveal any internal parameters when we explain the meaning of return values.

You also need a space in between `EOF` and `Exception`. Even it would be better to call it something like `"End of file exception"`.```

Reply:
modified